### PR TITLE
Add log package to integrations repo

### PR DESCRIPTION
**Pull request by ruflin**
I would like to get the log package out of the integrations repo. Instead of creating a separate repo I was hoping we could maintain this in the integrations repo for now. Is alpha the correct directory or should it go somewhere else?

### CODE REVIEW

**Comment by mtojek** [comment]
I think this is a good moment to involve team into this PR. /cc @exekias

File: dev/packages/alpha/log/0.1.0/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+version: 0.1.0
+categories: ["logs"]
+release: ga
+license: basic
+
+
+datasources:
+  - name: logs
+    title: Logs datasource
+    description: Datasource for your log files.
```
Comment (exekias) [inline dev/packages/alpha/log/0.1.0/manifest.yml:17]: I see datasource is mentioned in the title and description, isn't this implicit? How exposed are the users to this name in the UI?

Comment (ruflin) [reply dev/packages/alpha/log/0.1.0/manifest.yml:17]: yes, should be remove. Good time to clean this up.

File: magefile.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ var (
 	publicDir      = "./public"
 	buildDir       = "./build"
 	storageRepoDir = filepath.Join(buildDir, "package-storage")
-	packagePaths   = []string{filepath.Join(storageRepoDir, "packages"), "./dev/packages/beats/"}
+	packagePaths   = []string{filepath.Join(storageRepoDir, "packages"), "./dev/packages/beats/", "./dev/packages/alpha/"}
```
Comment (ruflin) [inline magefile.go:36]: @mtojek I added this to be able to test the log package. Should "new" packages go into a different directory?

Comment (mtojek) [reply magefile.go:36]: I think we should come back to cleaning things up here and also removing package versions (similarly to https://github.com/elastic/package-registry/pull/460 ). You can leave "alpha" directory here, but agree that a true clean up should be performed soon.

I didn't insist on performing it immediately as it's related to the layout of the repository.

Comment (ruflin) [reply magefile.go:36]: Agree. If you are ok with leaving it and ok with the PR, could you give a 👍 ?

Comment (mtojek) [reply magefile.go:36]: I think this is the next step on the roadmap - focus on the repository layout of integrations.

Comment (exekias) [reply magefile.go:36]: ++ I think we should target having the new layout soonish so we can start adding more integrations in a more definitive fashion

**Pull request merged**